include/xtensor/xaccumulator.hpp

```diff
@@ -68,6 +68,24 @@ namespace xt
         return accumulator_type(std::forward<RF>(accumulate_func), std::forward<IF>(init_func));
     }
 
+    template <std::size_t I, typename ...T>
+    decltype(auto) get(xaccumulator_functor<T...>&& v)
+    {
+        return std::get<I>(static_cast<std::tuple<T...>&&>(v));
+    }
+
+    template <std::size_t I, typename ...T>
+    decltype(auto) get(xaccumulator_functor<T...>& v)
+    {
+        return std::get<I>(static_cast<std::tuple<T...>&>(v));
+    }
+
+    template <std::size_t I, typename ...T>
+    decltype(auto) get(xaccumulator_functor<T...> const& v)
```

Comment:
Formatting: The convention is to put `const` before the type.

Reply:
Do you have a tool of choice to format the code?

Reply:
Not at the moment. Clang-format cannot be used for some of our conventions. However we have occasionaly been applying a clang format configuration manually, and reverted the changes in question (empty line after access modifiers in classes for example).

Reply:
Done, I think.